prettier.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: 'es5',
```

Comment:
probably ok to allow commas on the last line. since we are in a es6 anyway. 

it means fewer line changes when another entry is added